app/models/concerns/with_permissions.rb

```diff
@@ -1,24 +1,30 @@
 module WithPermissions
   extend ActiveSupport::Concern
 
+  included do
+    serialize :permissions, Mumukit::Auth::Permissions
+
+    validates_presence_of :permissions
+  end
+
   def student?
-    permissions.student? Organization.slug
+    mumukit_permissions.student? Organization.slug
   end
 
   def teacher?
-    permissions.teacher? Organization.slug
+    mumukit_permissions.teacher? Organization.slug
   end
 
   def set_permissions!(permissions)
     Mumukit::Auth::Store.set! self.uid, permissions
   end
 
-  def permissions
+  def mumukit_permissions
```

Comment:
Is there any difference in using `mumuki_permissions` instead of just permissions? I think here - I mean, just **in this line** - using the indirection of the store doesn't really help.